internal/hns/hnsendpoint.go

```diff
@@ -80,11 +92,26 @@ func HNSListEndpointRequest() ([]HNSEndpoint, error) {
 	return endpoint, nil
 }
 
+func HNSEndpointStatsRequest(id string) (*EndpointStats, error) {
```

Comment:
Comment on this as its exported

Reply:
We could probably leave this unexported also (unless I'm missing a caller other than the one in this same package)

Reply:
I was following the pattern with the other "request" functions.  Let me know what is perferred

Reply:
The reason the others are is because they expose a call at the root that just forwards to the /internal/hns method. We don't do that here so it should be fine to just unexport it. https://github.com/microsoft/hcsshim/blob/bf946774bacb7532d0bf09a47f19e4f6b37223cd/hnsendpoint.go#L32